Cargo.toml

```diff
@@ -11,6 +11,15 @@ keywords = ["string", "slice", "substring", "unicode", "utf-8"]
 categories = ["no-std", "algorithms", "parsing", "rust-patterns", "text-processing"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[dependencies]
+rayon_ = { package = "rayon", version = "1.10.0", optional = true }
```

Comment:
Because dependencies and features can conflict, so you can't name both rayon.

I personally think it's preferable for the user to have `rayon`

Reply:
```rust
[dependencies]
rayon = { ..., optional = true }

[features]
std = []
rayon = ["std"]
```

It should be possible to do something like this since Rust creates an implicit feature for optional dependencies.

In Rust 1.60+ you can also use the `dep:` syntax, but that requires bumping our MSRV